# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: da557d1806dd
# Full Hash: da557d1806dd9e1e519e00a4481b2e075f722ae8
# Author: Butkovits Atila <abutkovits@mozilla.com>
# Date: 2023-01-27 22:54 +0200
# Regressor Bug: 1788119
# File Overlap Count: 1
# Overlapping Files: Cargo.lock
# Description:
#   Backed out 5 changesets (bug 1788119) for causing bustages complaining about AVIFDecodedData. CLOSED TREE
# ==============================================================================

diff -r d980bebc6d26 -r da557d1806dd modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Fri Jan 27 20:35:34 2023 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Fri Jan 27 22:54:08 2023 +0200
@@ -7039,19 +7039,6 @@
   value: true
   mirror: always
 
-# Whether to allow decoding of animated AVIF sequences.
-- name: image.avif.sequence.enabled
-  type: RelaxedAtomicBool
-  value: false
-  mirror: always
-
-# Whether AVIF files containing sequences should be animated even when the
-# major brand is set to 'avif'.
-- name: image.avif.sequence.animate_avif_major_branded_images
-  type: RelaxedAtomicBool
-  value: false
-  mirror: always
-
 # Whether we attempt to decode JXL images or not.
 - name: image.jxl.enabled
   type: RelaxedAtomicBool